src/style.css

```diff
@@ -1,3 +1,95 @@
+:root {
+  --black: #000;
+  --grey: #666;
+  --yellow: #f0f4e7;
+  --violet: #e3d9ed;
+  --white: #fff;
+  --blue: #00acdc;
+  --main: 12px;
+  --line-height: 14px;
```

Comment:
Variable names should show their purpose, for example `--main-color`, etc. The color pallet could be changed in the future and you have to rename all occurrences if the main color has changed from red to blue.